--- ext-xdebug.ini.bak	2015-02-07 22:20:21.000000000 +0900
+++ ext-xdebug.ini	2015-02-08 03:30:14.000000000 +0900
@@ -1,2 +1,12 @@
 [xdebug]
 zend_extension="/usr/local/Cellar/php56-xdebug/2.2.7/xdebug.so"
+
+xdebug.remote_enable=1
+xdebug.remote_port="9001"
+xdebug.profiler_enable=1
+xdebug.profiler_output_dir="/tmp"
+xdebug.idekey = "PHPSTORM"
+xdebug.max_nesting_level=500
+xdebug.var_display_max_children=-1
+xdebug.var_display_max_data=-1
+xdebug.var_display_max_depth=-1
